src/layout/List/ListComponent.tsx

```diff
@@ -26,6 +26,7 @@ export const ListComponent = ({
   pagination,
   formData,
   handleDataChange,
+  getTextResourceAsString,
   sortableColumns,
   tableHeadersMobile,
```

Comment:
Is this not necessary for tableHeadersMobile as well?

Reply:
My thinking also immediately went there, but `tableHeadersMobile` specifies the headers to show on mobile, without giving you the option to change the header text. Have a look at the examples in the docs: https://docs.altinn.studio/app/development/ux/components/listcomponent/

Reply:
Aha, okay 👍